src/AddTodoForm.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+let AddTodoForm = () => {
```

Comment:
If we start implying that we can reassign component variables we can get into very sticky situations down the line. Once we assign a component to a variable it's best to secure that association.